source/agora/common/EnrollmentManager.d

```diff
@@ -517,72 +517,3 @@ unittest
     assert(!man.updateEnrolledHeight(utxo_hash, 9));
     assert(man.getEnrolledHeight(utxo_hash2) == 0);
 }
-
```

Comment:
I think we should keep this unittest too.

This one tests that `createEnrollment` makes a valid enrollment. And the unittest you added tests the validation function itself. Does that sound good to you?

Reply:
This retained the existing test and added a createEnrollment unittest.